daisy_workflows/export/disk_resizing_mon.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+METADATA_URL="http://metadata/computeMetadata/v1/instance"
+
+MAX_SIZE=$1
+BUFFER_SIZE=25
+INTERVAL=10
+
+echo "Max disk size ${MAX_SIZE}GB, buffer size ${BUFFER_SIZE}GB, starting monitoring available disk buffer every ${INTERVAL}s..."
+while [ 1 ]
+do
+  sleep ${INTERVAL}
+
+  # Check whether available buffer space is lower than threshold.
+  AVAILABLE_BUFFER=$(df -BG /dev/sdc --output=avail | sed -n 2p)
+  AVAILABLE_BUFFER=${AVAILABLE_BUFFER%?}
+  if [ ${AVAILABLE_BUFFER} -ge ${BUFFER_SIZE} ]; then
+    continue
+  fi
+
+  # Decide the new size, which won't be higher than max size.
+  CURRENT_SIZE=$(lsblk /dev/sdc --output=SIZE | sed -n 2p)
+  CURRENT_SIZE=${CURRENT_SIZE%?}
+  NEXT_SIZE=$(awk "BEGIN {print int(${CURRENT_SIZE} + ${BUFFER_SIZE})}")
+  if [ ${NEXT_SIZE} -gt ${MAX_SIZE} ]; then
```

Comment:
shouldn't you error on this condition?

Reply:
no, its purpose is to prevent unused extra space assignment. anyways I need to remove it because of the change of the design